src/core/utils/importEntrypointFile.ts

```diff
@@ -49,7 +49,6 @@ export async function importEntrypointFile<T>(
     cache: false,
     debug: config.debug,
     esmResolve: true,
-    interopDefault: true,
```

Comment:
Regression after switching to `esbuild` from `babel`: if there was no default export, this function returned an empty object instead of `undefined`.

Removing `interopDefault` fixes that.